feature/platform/fabric/otg_tests/sampled_backplane_capacity_counters_test/README.md

```diff
@@ -66,17 +48,19 @@ the available backplane capacity counters correctly while forwarding traffic
     *   /components/component/integrated-circuit/backplane-facing-capacity/state/total-operational-capacity
     *   /components/component/integrated-circuit/backplane-facing-capacity/state/available-pct
 
-*   Disable one of the available FABRIC component
+*   Make sure we recieve the initial metric for each of the telemetry paths
+
+*   Disable two FABRIC components
 
     *   set /components/component/{fabric}/config/power-admin-state to POWER_DISABLED
 
-*   Validate that we recieve changed metrics of a lower value for each of the telemetry paths
+*   Validate that we recieve changed metric of a lower value for each of the telemetry paths
```

Comment:
Only these should change:
    *   /components/component/integrated-circuit/backplane-facing-capacity/state/total-operational-capacity
    *   /components/component/integrated-circuit/backplane-facing-capacity/state/available-pct